packages/field-content/src/blocks/heading.js

```diff
@@ -0,0 +1,14 @@
+import { importView } from '@keystone-alpha/build-field-types';
+import Block from '../Block';
+
+export default class HeadingBlock extends Block {
+  get type() {
+    return 'heading';
+  }
+  getAdminViews({ availableBlocks }) {
+    return [
+      importView('../views/editor/blocks/heading'),
+      ...new availableBlocks.paragraph().getAdminViews({ availableBlocks }),
```

Comment:
Here we consume the dependency injected blocks, and further pass it on to the block we're depending upon. Dependency-ception!